README.md

```diff
@@ -19,7 +22,35 @@ The easiest way to get started locally is simply [get yourself a copy of open so
 $ java -jar joshsim.jar run simulation.josh
 ```
 
-Simply run the jar without any command specified to get further help documentation. You can also specify an output location
+While COGs, geotiffs, and netCDF files can be provided directly, the preferred approach is to provide a jshd file which preprocesses these geospatial inputs for speed:
+
+```
+$ java -jar joshsim.jar preprocess simulation.josh MySimulation data.nc variable units output.jshd
+$ java -jar joshsim.jar run simulation.josh --data output.jshd
+```
+
+Available commands include:
+
+- `validate` for checking syntax:
+  ```
+  $ java -jar joshsim.jar validate simulation.josh
+  ```
+- `run` for executing simulations:
+  ```
+  $ java -jar joshsim.jar run simulation.josh
+  ```
+- `server` for starting a local web interface:
+  ```
+  $ java -jar joshsim.jar server
+  ```
+- `preprocess` for creating optimized jshd files:
+  ```
+  $ java -jar joshsim.jar preprocess simulation.josh MySimulation data.nc variable units output.jshd
+  ```
+
+Simply run the jar without any command specified to get further help documentation.
+
+You can also specify an output location:
```

Comment:
Let’s go ahead and get rid of the output location example. We will document that in more detail in a later PR.